deploy/kubernetes/simplyblock-storagenode.yaml

```diff
@@ -0,0 +1,117 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: storage-node-sa
+
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: storage-node-role
+rules:
+- apiGroups: [""]
+  resources: ["pods", "namespaces", "pods/exec"]
+  verbs: ["list", "get", "create", "delete"]
+- apiGroups: ["apps"]
+  resources: ["deployments"]
+  verbs: ["create", "delete"]
+
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pods-list-sn
+subjects:
+- kind: ServiceAccount
+  name: storage-node-sa
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: storage-node-role
+  apiGroup: rbac.authorization.k8s.io
+
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: simplyblock-storages
+  annotations:
+    helm.sh/hook: post-install
+spec:
+  selector:
+    matchLabels:
+      app: simplyblock-storages
+  template:
+    metadata:
+      labels:
+        app: simplyblock-storages
+    spec:
+      serviceAccountName: storage-node-sa
+      nodeSelector:
```

Comment:
How do you handle for cases of secondary storage node and also node to perform primary node restart on as this two have a separate node label

Reply:
The purpose of this folder (`deploy/kubernetes`) is to provide examples. So of all the 3 storage node type its best to have example for `simplyblock-storage-plane` and remaining can be re-created from this file by just modifying the app Labels and nodeSelector

Reply:
Won't it be best we just created them also in the example?

Reply:
now that we don't have a secondary storage nodes, I think having just file should be sufficient I guess